### DISCUSSION TITLE
Posts are not rendering Template output

### DISCUSSION THREAD

**Discussion by maw246** [chipzoller/hugo-clarity#379] [Q&A] [answered]
I've started using the Clarity theme, and even the starter sample site is showing this behavior: when I write a Go render template into a post, it prints the literal text of my formula.  For example, entering `{{ add 1 2 }}` in my _*post/mybundle/index.md*_ in the file bundle renders into the browser as `{{ add 1 +2 }}`:
![image](https://user-images.githubusercontent.com/5851873/190089225-f5993884-8a5a-4392-8033-ef4d58741a30.png)

What would cause this? What might I be doing wrong?

**Comment by rootwork** [comment, marked as answer]
Maybe I'm not fully understanding what you're asking, but I don't believe Hugo supports putting Go template tags (beyond Hugo-defined shortcodes) into Markdown content files. The only thing that gets interpreted in a Markdown file is valid Markdown (including HTML) + shortcodes.

**Reply by maw246** [reply]
I think you're understanding correctly. I'm inexperienced with Hugo, and thought I could write Go template tags into "content" files. It was a late night, and I was struggling. And additional research today led me to understand that template tags in content don't seem to be an option. (I didn't see anywhere in the Hugo docs that explicitly told me that, but... I sorta picked up on it after reading enough different docs pages.) Thanks for responding to what I now realize was a rather uninformed question from me :)
